Bottom: f43c6cebc504ae1a926f2fdaf5e0b48d452a34db
Top:    a26e66482bdf75f77d3034f2d5813960b6b04cc8
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-10 09:37:49 +0800

Refresh of autofs-5.0.6-fix-compilation-of-lookup_ldap_c-without-sasl.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 325ed15..f6acea2 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -53,8 +53,6 @@
 - fix probe each nfs version in turn for singleton mounts.
 - misc man page fixes.
 - fix add null check in parse_server_string().
-- fix getautomntbyname_r() declaration.
-- fix compilation of defaults.c without openldap.
 - fix compilation of lookup_ldap.c without sasl.
 
 25/07/2012 autofs-5.0.7
